sklearn/inspection/_permutation_importance.py

```diff
@@ -155,7 +190,14 @@ def permutation_importance(
     sample_weight : array-like of shape (n_samples,), default=None
         Sample weights used in scoring.
 
-        .. versionadded:: 0.24
+    max_samples : int or float, default=1.0
+        The number of samples to draw from X to compute feature importance in each repeat (without
+        replacement).
+        - If int, then draw `max_samples` samples.
+        - If float, then draw `max_samples * X.shape[0]` samples.
+        No effect if max_samples is equal to 1.0 or X.shape[0].
+
+        .. versionadded:: 1.0.dev0
```

Comment:
The release version is referenced usually.

```suggestion
        .. versionadded:: 1.0
```

Reply:
Thanks! No there is no reference to this kind of sub-sampling that I am aware of other than what I mentioned in the PR (the implementation in a corresponding R package)

Reply:
The docstring used more than 88 lines before, so I am not able to fix this without affecting other parts of the documentation